parity-util-mem/Cargo.toml

```diff
@@ -46,6 +47,8 @@ jemalloc-global = ["jemallocator"]
 # use mimalloc as global allocator
 mimalloc-global = ["mimallocator", "mimalloc-sys"]
 # implement additional types
-ethereum-impls = ["ethereum-types"]
+ethereum-impls = ["ethereum-types", "primitives"]
 # Full estimate: no call to allocator
 estimate-heapsize = []
+# primitives
+primitives = ["primitive-types"]
```

Comment:
any reason not to just use `primitive-types` as a feature, is the plan to add more dependencies under this feature? otherwise, `primitive-types` seems more explicit and doesn't require additional entry in `[features]` section

Reply:
Hmm I didn't know I could do it, let me check